src/emulator/gui/src/game_selector.cpp

```diff
@@ -39,7 +42,7 @@ void draw_game_selector(GuiState &gui, HostState &host) {
     ImGui::Begin("Game Selector", nullptr, ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoBringToFrontOnFocus | ImGuiWindowFlags_NoSavedSettings);
 
     if (gui.current_background) {
-        ImGui::GetBackgroundDrawList()->AddImage(reinterpret_cast<void *>(gui.current_background),
+        ImGui::GetBackgroundDrawList()->AddImage(reinterpret_cast<void *>(static_cast<uintptr_t>(gui.current_background)),
```

Comment:
`.get()` instead of `static_cast<>` here.

Reply:
try but no understund what you want said

Reply:
i can't use get here, no works.

Reply:
Yes I appear to have fooled myself.

Reply:
i see haha